content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -74,15 +74,15 @@ The available AccountSet flags are:
 | Flag Name                         | Decimal Value | Corresponding Ledger Flag         | Description   |
 |:----------------------------------|:--------------|:----------------------------------|:--------------|
 | `asfAccountTxnID`                 | 5             | (None)                            | Track the ID of this account's most recent transaction. Required for [`AccountTxnID`](transaction-common-fields.html#accounttxnid) |
-| `asfAuthorizedNFTokenMinter`      | 10            | (None)                            | Enable to allow another account to mint non-fungible tokens (NFTokens) on this account's behalf. Specify the authorized account in the `NFTokenMinter` field of the [AccountRoot](accountroot.html) object. _(Added by the [NonFungibleTokensV1_1 amendment][].)_ |
+| `asfAuthorizedNFTokenMinter`      | 10            | (None)                            | Enable to allow another account to mint non-fungible tokens (NFTokens) on this account's behalf. Specify the authorized account in the `NFTokenMinter` field of the [AccountRoot](accountroot.html) object. To remove an authorized minter, enable this flag and omit the `NFTokenMinter` field. _(Added by the [NonFungibleTokensV1_1 amendment][].)_ |
```

Comment:
The usage of this flag to remove an authorized minter is mentioned at the bottom of the page, but I thought it would make sense to repeat here in the description of the flag, especially since that's the whole reason this flag exists. (The protocol doesn't have another way to tell the difference between "don't change the field" and "remove the field".)